### ISSUE TITLE
Compilation does not work

### ISSUE THREAD

**Issue by stephanschubert** [Rahlir/OmniFocusScripts#1]
I tried the described methods but none work:

```
./compile.sh Library
rm: cannot remove '/Users/sschubert/Library/Script Libraries/omnifocuslib.scpt': No such file or directory
Library/omnifocuslib.applescript:115: error: Expected end of line but found identifier. (-2741)
```

```
./compile.sh Library/omnifocuslib.applescript
rm: cannot remove '/Users/sschubert/Library/Application Scripts/com.omnigroup.OmniFocus3/libraryomnifocuslib.applescript.scpt': No such file or directory
osacompile: couldn't read file Libraryomnifocuslib.applescript/libraryomnifocuslib.applescript.applescript: No such file or directory
```

And manually:

```
osacompile -o Library/omnifocuslib.scpt Library/omnifocuslib.applescript
Library/omnifocuslib.applescript:115: error: Expected end of line but found identifier. (-2741)
```

Any ideas?

**Comment by Rahlir** [comment]
Yeah, the first and the third attempt are the correct ones and they throw the same error it appears. (The "cannot remove" error is not important, it just means that there is no prior version installed, I will fix the script so that it doesn't show this). I am assuming you don't have the app Timing on your computer and that's most likely why the compilation doesn't work. I will look into fixing it tonight.

**Comment by Rahlir** [comment]
Sorry for the delay, I was really busy. The fix was implemented in b99e4a6c91829685498a132bec3c2bab28bd290b
 
I have changed the compile script so that running it shouldn't cause the issue with Timing app not being installed anymore. The timing app library is separate now and to compile it the script should be run with the -t option:
```bash
./compile.sh -t Library
```
Running the script without it as 
```bash
./compile.sh Library
```
should work for those that do not have Timing on their computer and do not want to use the `Start Task` script.

I have no easy way to reproduce this bug and test the fix, but I am fairly confident this fix should work, so I am closing the issue for now. If the issue persists, let me know.
